agent/engine/docker_container_engine.go

```diff
@@ -726,7 +726,7 @@ func (dg *dockerGoClient) Stats(id string, ctx context.Context) (<-chan *docker.
 	go func() {
 		statsErr := client.Stats(options)
 		if statsErr != nil {
-			seelog.Warnf("Error retrieving stats for container %s: %v", id, statsErr)
+			seelog.Debugf("Error retrieving stats for container %s: %v", id, statsErr)
```

Comment:
Would it be reasonable to add a check here, if we keep receiving the error of "no such container" here, can we check the container status and if the container has already stopped, we just close the stream?